rpp32-fec-hedgehog/fec-atelier
# QA Framework

This PR implements the following:
- QA (primary component)
- 4 child components: QAnswer, QASearch, QAsk, and QAView
- Server route for Q&A component
- API call made from server (successfully getting data all the way back to client)

I added separators in the server and apiCalls files just to keep things a bit more organized. Please let me know if y'all want me to get rid of/change these.

Lastly, I couldn't get ajaxPrefilter to work for the life of me, so I changed my config.js and the example file to match it. Please let me know if y'all are doing something similar/different, we might want to discuss configs when we next meet just to make sure we're all on the same page with them. This way, we can set up a single API_KEY variable in the apiCalls file that everyone can use.

Please let me know what y'all think and if there are any changes you'd like me to make.
